api/testEvents.yaml

```diff
@@ -0,0 +1,56 @@
+version: 3.0.0
+components:
+  schemas:
+    TestRunEvent:
+      type: object
+      properties:
+        type:
+          type: string
+        stage:
+          type: string
+          enum:
+            - trigger
+            - trace
+            - test
+        description:
+          type: string
+        createdAt:
+          type: string
+          format: date-time
+        testId:
+          type: string
+        runId:
+          type: string
+        dataStoreConnection:
+          $ref: "./config.yaml#/components/schemas/ConnectionResult"
+        polling:
+          $ref: "#/components/schemas/PollingInfo"
+        outputs:
+          type: array
+          items:
+            $ref: "#/components/schemas/OutputInfo"
+
+    PollingInfo:
+      type: object
+      properties:
+        numberSpans:
+          type: integer
+        numberIterations:
+          type: integer
+        reasonNextIteration:
+          type: string
+        isComplete:
+          type: boolean
+
+    OutputInfo:
+      type: object
+      properties:
+        logLevel:
+          type: string
+          enum:
+            - warning
+            - error
```

Comment:
Should we include a `info` type?

Reply:
I don't think it's worth unless we want to show in the event log what are the values for each output after they were resolved. @xoscar what do you think?

Reply:
I think the info type is not something we'll be using in the near future so I think its fine to not have it

Reply:
Yeah, makes sense @mathnogueira